### ISSUE TITLE
Deeplab Training Detail

### ISSUE THREAD

**Issue by donhk1011** [msracver/Deformable-ConvNets#271]
Hi,

I could not reproduce the deeplab baseline on cityscapes. The baseline is getting ~66% lower than 70.4% in the paper.

I found there is cropping in this repo, https://github.com/msracver/Deformable-ConvNets/blob/master/experiments/deeplab/cfgs/deeplab_resnet_v1_101_cityscapes_segmentation_base.yaml#L60)
But, in the paper, it seems to use the whole image as input in training time.

In addition, the learning rate is different. (5e-4 (https://github.com/msracver/Deformable-ConvNets/blob/master/experiments/deeplab/cfgs/deeplab_resnet_v1_101_cityscapes_segmentation_base.yaml#L52) vs 1e-3 in the paper)

Could you please specify the differences between this repo and the paper? 

And is it possible to reproduce the number of the deeplab baseline with the same configuration of this repo?

Thanks!
